lms/djangoapps/courseware/views/views.py

```diff
@@ -1298,6 +1298,13 @@ def financial_assistance_form(request):
             mode_slug=CourseMode.VERIFIED
         ).exists()
     ]
+    annual_incomes = [
+        {'name': _('Less than $5,000'), 'value': 'Less than $5,000'},
```

Comment:
Nitpick: I'd encourage you to assign these strings to variables/constants to reduce duplication.